src/test/java/uk/gov/companieshouse/confirmationstatementapi/configuration/InterceptorConfigRouteMatchingTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.companieshouse.confirmationstatementapi.configuration;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.servlet.HandlerExecutionChain;
+import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest()
+class InterceptorConfigRouteMatchingTest {
+
+    @Autowired
+    private RequestMappingHandlerMapping mapping;
+
+    @Test
+    void interceptorsMatchIntendedRoutesTest() throws Exception {
```

Comment:
nice test :-)

Reply:
Thanks!

Reply:
One thought, I might have included a path for each of the controllers present in this API, i.e. the new registered-email-address path is not tested, etc.

Reply:
This was the intention, e.g. registered-email-address path test case is defined on line 56-7.

Reply:
oops, sorry